src/features/game/mode/move/GameModeMove.ts

```diff
@@ -79,16 +81,16 @@ class GameModeMove implements IGameUI {
     });
 
     moveMenuContainer.add(
-      buttons.map((button, index) =>
-        this.createMoveButton(
-          button.text,
+      buttons.map((button, index) => {
+        return this.createMoveButton(
+          '[ ' + (index + 1) + ' ]  ' + button.text,
```

Comment:
Suggestion to convert this into a separate method :+1:

Reply:
A new method ```createButtonText``` is created under ButtonUtils to handle the displaying of the text for buttons with indices.